### DISCUSSION TITLE
How to get image from pocketbase to my next project

### DISCUSSION THREAD

**Discussion by Ebibeck** [vercel/next.js#49099] [Help] [answered]
How can i get image from pocketbase to my next project

**Comment by LimSB-dev** [comment]
In your Next.js project, you can use the Image component from the Next.js built-in 'next/image' module to display the image.

Import the Image component from 'next/image' and use it to display the image.
Pass the image URL as the src prop to the Image component.

```javascript
import Image from 'next/image';

function YourComponent() {
  return (
    <div>
      <Image
        src="https://your-pocketbase-image-url.com/image.jpg"
        alt="description of the image"
        width={500}
        height={500}
      />
    </div>
  );
}
```

**Reply by Ebibeck** [reply]
Thank you.... 😊😊

**Reply by Ebibeck** [reply]
Wait what about the next. Config

**Comment by LimSB-dev** [comment]
Oh! I forgot that part.

I think you can do it like this.

```js
module.exports = {
  images: {
    domains: ['your-pocketbase-domain.com'],
  },
  ...
};
```

If it was helpful, I would appreciate it if you could press the answer button.

**Reply by Ebibeck** [reply, marked as answer]
Ok thanks again.... So much help...
